defs/bitcoin/stakenet.json

```diff
@@ -0,0 +1,41 @@
+{
+    "coin_name": "Stakenet",
+    "coin_shortcut": "XSN",
+    "coin_label": "Stakenet",
+    "website": "https://stakenet.io",
+    "github": "https://github.com/X9Developers/XSN",
+    "maintainer": "<support@stakenet.io>",
```

Comment:
maintainer has to be a real person, who is fully responsible for maintaining this definition factually correct at all times, not just some email alias

Reply:
it is updated.